### DISCUSSION TITLE
Directus on Plesk with npm-version > 6

### DISCUSSION THREAD

**Discussion by FelixPopp** [directus/directus#12380] [Archive]
I've tried to install directus on Plesk with your guide from [here](https://docs.directus.io/getting-started/installation/plesk/).
It fails because "node was not found", although I've included the `scripts-prepend-node-path=true`. 

The reason is, that I'm using the node.js LTS-Version 16 (npm=8) and since npm-version 7 the option `scripts-prepend-node-path` was removed. See npm-Ticket [here](https://github.com/npm/cli/issues/2808#issuecomment-821431605).

Do you know any workaround because node.js 14 (npm=6) will be outdated in a year?

**Comment by rijkvanzanten** [comment]
I don't use Plesk myself, so I don't know what witchcraft is going on under the hood there, but maybe instead of relying on `scripts-prepend-node-path`, you could update the script in package.json from `"bootstrap": "directus bootstrap"` to `"bootstrap": "/full/path/to/node_modules/directus bootstrap"` ?

**Reply by FelixPopp** [reply]
The error already appears on npm install, sorry I should have made that clear.
I will try npm install locally without Plesk and then upload the node_modules to Plesk. Maybe this will work.

**Reply by rijkvanzanten** [reply]
I see! In that case, this should be a problem for effectively any Node based project on Plesk right? Not just Directus? 🤔
